src/Processing.js

```diff
@@ -4066,10 +4066,36 @@
     p.noLoop = function() {
       doLoop = false;
       loopStarted = false;
-      clearInterval(looping);
+      
+      looping = false;
       curSketch.onPause();
     };
 
+requestAnimationFrame= (function(){
+      return window.requestAnimationFrame ||
+             window.webkitRequestAnimationFrame ||
+             window.mozRequestAnimationFrame    ||
+             function( callback ){
+                var _h= function(){callback(Date.now());};
+                window.setTimeout(_h, curMsPerFrame);
+             };
+     })();
+
+    var _now, _then=0, _doDraw = function(t) {
+      requestAnimationFrame(_doDraw);
+        if (!looping) { return; }
+        _now= t;
```

Comment:
`t` is invariant inside this function (at least until we draw a new frame), do we need to alias it at all?